packages/styles/scss/components/date-picker/_flatpickr.scss

```
@@ -423,29 +423,29 @@
     height: rem(40px);
     align-items: center;
     justify-content: center;
-    color: $text-01;
+    color: $text-primary;
     cursor: pointer;
     transition: all $duration-fast-01 motion(standard, productive);
 
     &:hover {
-      background: $hover-ui;
+      background: $background-hover;
     }
 
     &:focus {
       @include focus-outline('outline');
 
-      outline-color: $interactive-01;
+      outline-color: $background-brand;
```

Same thing here since I believe this is a button?

```suggestion
      outline-color: $button-primary;
```